### PR TITLE
always more e2e tests

### PR DESCRIPTION
**Pull request by ludydoo**

### CODE REVIEW

File: fleetshard/pkg/central/operator/upgrade.go [hunk 1]
```diff
@@ -17,7 +17,7 @@ const (
 	// ACSOperatorNamespace default Operator Namespace
 	ACSOperatorNamespace = "rhacs"
 	// ACSOperatorConfigMap name for configMap with operator deployment configurations
-	ACSOperatorConfigMap     = "operator-config"
+	ACSOperatorConfigMap     = "fleet-manager-gitops-config"
```
Comment (SimonBaeumer) [inline fleetshard/pkg/central/operator/upgrade.go:20]: Can be deleted

**Comment by ludydoo** [comment]
old

**Pull request closed without merging**